src/Process.php

```diff
@@ -164,7 +164,7 @@ public function __construct($cmd, $cwd = null, array $env = null, array $fds = n
      * @param float          $interval    Interval to periodically monitor process state (seconds)
      * @throws \RuntimeException If the process is already running or fails to start
      */
-    public function start(LoopInterface $loop = null, $interval = 0.1)
+    public function start(?LoopInterface $loop = null, $interval = 0.1)
```

Comment:
Could you update this method the same way this method has been updated in another package? https://github.com/reactphp/socket/pull/318/files#diff-2c0b16424077d1ecafdbbca23aa3f137bc007241b00fe55dd536434c147326ca This way we can ensure it runs on all PHP versions in rage